addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -854,6 +854,7 @@ const Wysiwyg = Widget.extend({
      * @returns {String}
      */
     getValue: function (options) {
+        this.odooEditor && this.odooEditor.cleanForSave(this.$editable[0]);
```

Comment:
We should not mutate the editable when getting a value.

Reply:
I agree, this was done for development, in order to see the changes applied to the saved document. It revealed important discrepancies that should be addressed. The sanitization over the whole document should happen before we arrive in getValue.